examples/stable_diffusion_v2/ldm/modules/diffusionmodules/model.py

```diff
@@ -265,7 +265,8 @@ def __init__(self, *, ch, out_ch, ch_mult=(1,2,4,8), num_res_blocks,
             down = nn.Cell()
             down.block = block
             down.attn = attn
-            down.downsample = downsample
+            if i_level != self.num_resolutions-1:
+                down.downsample = downsample
```

Comment:
Require checkpoint conversion. and re-run the test for infer/train for ensure they still work well after  modifying this architecture

Reply:
we are changing back to the original arch, consistent with pretrained models